eel/src/storage_persister.rs

```diff
@@ -447,13 +457,26 @@ impl<ChannelSigner: Sign> Persist<ChannelSigner> for StoragePersister {
 
 fn sync_persist_monitor_remotely(
     storage: Arc<Box<dyn RemoteStorage>>,
+    encryption_key: &[u8; 32],
     chain_monitor: Arc<ChainMonitor>,
     data: Vec<u8>,
     channel_id: OutPoint,
 ) {
     let key = channel_id.to_channel_id().to_hex();
+    let encrypted_data = match encrypt(&data, &encryption_key) {
+        Ok(data) => data,
+        Err(e) => {
+            error!("Failed to encrypt the ChannelMonitor: {}", e.to_string());
+            return;
+        }
```

Comment:
Here it makes sense to return an error and make `persist_new_channel()` return `ChannelMonitorUpdateStatus::PermanentFailure` in case `sync_persist_monitor_remotely` fails. I've already made this change on PR #204, so let's **not** do it here. When we rebase either this PR or the other, we fix it.

Reply:
I'm not sure if I understand you correctly. Rebasing onto #204 forced me to make a change here anyways, so could you please have a look at it again?

Reply:
Forget that, you did exactly what I meant. Only fix after rebasing, not before :)